src/diffusers/utils/remote_utils.py

```diff
@@ -0,0 +1,127 @@
+import base64
+import io
+import json
+from typing import List, Literal, Optional, Union, cast
+
+import requests
+
+from .import_utils import is_safetensors_available, is_torch_available
+
+
+if is_torch_available():
+    import torch
+
+    from ..image_processor import VaeImageProcessor
+    from ..video_processor import VideoProcessor
+
+    if is_safetensors_available():
+        import safetensors
+
+    DTYPE_MAP = {
+        "float16": torch.float16,
+        "float32": torch.float32,
+        "bfloat16": torch.bfloat16,
+        "uint8": torch.uint8,
+    }
+
+
+from PIL import Image
+
+
+def remote_decode(
+    endpoint: str,
+    tensor: "torch.Tensor",
+    processor: Optional[Union["VaeImageProcessor", "VideoProcessor"]] = None,
+    do_scaling: bool = True,
+    output_type: Literal["mp4", "pil", "pt"] = "pil",
+    image_format: Literal["png", "jpg"] = "jpg",
+    partial_postprocess: bool = False,
+    input_tensor_type: Literal["base64", "binary"] = "base64",
+    output_tensor_type: Literal["base64", "binary"] = "base64",
```

Comment:
`output_type` and `output_tensor_type` appear to be a bit confusing in terms of naming. I think we could distinguish better. 

Perhaps `remote_output_type`?

Reply:
Yeah I mentioned I'm going to do this
https://huggingface.slack.com/archives/C065E480NN9/p1740466217754749?thread_ts=1740463081.487609&cid=C065E480NN9